source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/events.py

```diff
@@ -1004,3 +1004,46 @@ def _randomize_prop_by_op(
             f"Unknown operation: '{operation}' for property randomization. Please use 'add', 'scale', or 'abs'."
         )
     return data
+
+def randomize_body_coms(
```

Comment:
Could you please place this after: randomize_rigid_body_mass function to keep ordering consistent to follow?